### PR TITLE
Add HAML to list of supported files

### PR DESCRIPTION
**Pull request by theparadoxer02**
worked on https://github.com/coala/coala-bears/issues/2169

### CODE REVIEW

**Comment by RaiVaibhav** [comment]
>Please follow the commit guidelines at  

:+1: 

unack 3ad09f1b2795423e607cff921321272d6c3659e3

**Comment by theparadoxer02** [comment]
Modified the commit's comment now @RaiVaibhav @ishanSrt

**Comment by ishanSrt** [comment]
I don't think the commit messages require any kind of markup so you should remove the backticks

**Comment by theparadoxer02** [comment]
@ishanSrt done

**Comment by ishanSrt** [comment]
You will have to ping a moderator for AppVeyour build failed

**Comment by RaiVaibhav** [comment]
LGTM :)
Please take up a `low` issue, currently your PR gets blocked by https://github.com/coala/coala-bears/issues/2175

**Comment by newbazz** [comment]
You can rebuild ur PR. ;)

**Comment by RaiVaibhav** [comment]
>You can rebuild ur PR. ;)

He main to say Rebase your PR :stuck_out_tongue:

**Comment by aashraybhandar1** [comment]
@theparadoxer02  rebase your commit ... see git basics to see how to rebase your commits .. the CI has been fixed now

**Comment by theparadoxer02** [comment]
I just clicked on `Update Branch`option, is the process complete or i still need to make some changes ?

**Comment by shreyateeza** [comment]
@theparadoxer02 Correct the comment in this way.
image
As given in the image check the items in the checklist.

**Comment by RaiVaibhav** [comment]
@shreyateeza  although we made a PR template but. No need if the assignee understands what need to be done next

@theparadoxer02  you changed the Head of commit , change it to 
>README.rst: Add HAML to supported language list

`git commit --amend` and  `force push` will help you

**Comment by theparadoxer02** [comment]
@RaiVaibhav @shreyateeza after i clicked on `Update Branch` i realized that it append a git commit, i rebased my branch locally and pushed, now it has appended the other commit made in master branch with my commit is at the top. Is it fine now ?

**Comment by RaiVaibhav** [comment]
@theparadoxer02 your master branch is  still not updated, your branch is updated
what you can do now to change the `Commit Head` is 
1.) `git checkout issue2169`
2.) `git commit --amned`
change the head of commit from `README.rst: Modify supported language list` to `README.rst: Add HAML to Language Supported` or something like that and so on add  others things in commit part.
3.) `git push origin issue2169 --force`

Hoping you have got the `rebase` part.

**Comment by theparadoxer02** [comment]
@RaiVaibhav  done

**Comment by RaiVaibhav** [comment]
ack cb44bf4697094eaf1dd3a1041146963c64a6d3b8

**Comment by theparadoxer02** [comment]
Thanks  everyone for your support. I am looking into a low issue now

File: README.rst [hunk 1]
```diff
@@ -87,7 +87,7 @@ To see what coala can do for your language, run:
 +----------------------------+----------------------------+----------------------------+
 | Go                         | Python 3                   | YAML                       |
 +----------------------------+----------------------------+----------------------------+
-| Haskell                    | R                          |                            |
+| Haskell                    | R                          | HAML                       |
```
Comment (Makman2) [inline README.rst]: Doesn't keep the alphabetical order ;)
(I know this layout is anyway crappy to maintain... I think we got an issue to do it differently.)

**Comment by Makman2** [comment]
unack cb44bf4

**Comment by Makman2** [comment]
unack cb44bf4

File: README.rst (hunk 1 above)
Comment (theparadoxer02) [reply README.rst]: Do i have to rearrange it ?

**Pull request closed without merging**